scripts/generators/extension_helper_generator.py

```diff
@@ -92,11 +138,11 @@ class APIVersion {
         return *this;
     }
 
-    bool valid() const { return api_version_ != VVL_UNRECOGNIZED_API_VERSION; }
-    uint32_t value() const { return api_version_; }
-    uint32_t major() const { return VK_API_VERSION_MAJOR(api_version_); }
-    uint32_t minor() const { return VK_API_VERSION_MINOR(api_version_); }
-    uint32_t patch() const { return VK_API_VERSION_PATCH(api_version_); }
+    bool Valid() const { return api_version_ != VVL_UNRECOGNIZED_API_VERSION; }
+    uint32_t Value() const { return api_version_; }
+    uint32_t Major() const { return VK_API_VERSION_MAJOR(api_version_); }
+    uint32_t Minor() const { return VK_API_VERSION_MINOR(api_version_); }
+    uint32_t Patch() const { return VK_API_VERSION_PATCH(api_version_); }
```

Comment:
why do these need to be changed?

Reply:
These were needed because on QNX the STL headers include some platform headers that #define major and minor as macros and there was no trivial way to avoid including those platform headers, so we switched to CamelCase names instead. This change has already been upstreamed in the meantime, it just happened since the last downstreaming.

Reply:
Ah right - QNX.  If it's already been upstreamed - that's great too.